File: src/main/java/com/official/memento/todo/domain/vo/PrioritizedToDo.java
package com.official.memento.todo.domain.vo;

import java.time.LocalDate;

public record PrioritizedToDo(
        String task,
        Long id,
        LocalDate createdDate,
        LocalDate deadline,
        Float priority,
        Float urgency,
        Float importance,
        Double order
) {
}

File: src/main/kotlin/com/official/memento/todo/domain/vo/PrioritizedToDo.kt
package com.official.memento.todo.domain.vo

import java.time.LocalDate

data class PrioritizedToDo(
    val task: String,
    val id: Long,
    val createdDate: LocalDate,
    val deadline: LocalDate,
    val priority: Float,
    val urgency: Float,
    val importance: Float,
    val order: Int,
)
